src/db/Rating/rating.ts

```diff
@@ -14,7 +22,7 @@ export async function addRating(ratingToAdd: Rating): Promise<Rating> {
 	});
 	if (ratingExists) {
 		await _connection.close();
-		throw "User cannot add more than one rating to the same place";
+		throw new DbOperationError("User cannot add more than one rating to the same place");
```

Comment:
I like the custom error types a lot, good stuff!

Reply:
thank you!